importing assert from console in dateToString causes usage with Webpack v5 to break

Using mingo v6+ in an app built with webpack throws an error: `Module not found: Error: Can't resolve 'console' in...`. 

Webpack also includes the message: `BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.`

This is caused by importing assert from console in `dateToString.ts` and can be fixed by simply calling `console.assert` directly. A pull request for this is coming shortly.

--- a/src/operators/expression/date/dateToString.ts
+++ b/src/operators/expression/date/dateToString.ts
@@ -1,7 +1,5 @@
 // Date Expression Operators: https://docs.mongodb.com/manual/reference/operator/aggregation/#date-expression-operators
 
-import { assert } from "console";
-
 import { computeValue, Options } from "../../../core";
 import { AnyVal, Callback, RawObject } from "../../../types";
 import { isNil, isObject } from "../../../util";
@@ -97,7 +95,7 @@ export function $dateToString(
       } else if (props.name === "minuteOffset") {
         value = minuteOffset.toString();
       } else {
-        assert(
+        console.assert(
           !!operatorFn,
           `unsupported date format specifier '${formatSpecifier}'`
         );

--- a/src/operators/expression/date/dateToString.ts
+++ b/src/operators/expression/date/dateToString.ts
@@ -2,7 +2,7 @@
 
 import { computeValue, Options } from "../../../core";
 import { AnyVal, Callback, RawObject } from "../../../types";
-import { isNil, isObject } from "../../../util";
+import { assert, isNil, isObject } from "../../../util";
 import {
   adjustDate,
   computeDate,
@@ -95,7 +95,7 @@ export function $dateToString(
       } else if (props.name === "minuteOffset") {
         value = minuteOffset.toString();
       } else {
-        console.assert(
+        assert(
           !!operatorFn,
           `unsupported date format specifier '${formatSpecifier}'`
         );

